incidents.md

```diff
@@ -0,0 +1,110 @@
+Zettle API Incidents
+=====================
+# 30 March, 2021
+## (status: resolved) Finance API returned wrong `originatorTransactionType` 
+The following Finance API endpoint returned wrong `originatorTransactionType`:
+
+`GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/transactions`.
+
+<details><!-- start tag of the incident section-->
+<summary>Click to see the incident.</summary>
+    <h3>Incident summary</h3>
+        <p>The <code>originatorTransactionType</code> field was returned with wrong values.</p>
+        <p>The following table shows the expected and returned values for the field.<p>
+        <table style="text-align:left">
+            <thead>
+                <tr>
+                  <th>Expected <code>originatorTransactionType</code></th>
+                  <th>Returned <code>originatorTransactionType</code></th>
+                </tr>
+            </thead>
+                <tbody>             
+                <tr>
+                   <td>CARD_PAYMENT</td>
+                   <td>PAYMENT</td>
+                </tr>
+                <tr>
+                   <td>CARD_PAYMENT_FEE</td>
+                   <td>PAYMENT</td>
+                </tr>
+                <tr>
+                   <td>CARD_REFUND</td>
+                   <td>PAYMENT</td>
+                </tr>
+                <tr>
+                   <td>CARD_PAYMENT_FEE_REFUND</td>
+                   <td>PAYMENT</td>
+                </tr>        
+                </tbody>
+        </table>
+
+<h4 name="incidentDuration">Incident duration</h4>
+<p>Transactions made between the following timestamps were affected:</p>
+
+|Start time | End time |Total duration
+|:---- |:---- |:----
+|2021-03-30 16:03:11.437237 UTC |2021-03-30 16:59:45.00856  UTC |Approximate 56 minutes
+
+
+<h3>What do you need to do</h3>
+<ol>
+    <li><p>Refetch transactions that happened during the <a href="incidentDuration">incident duration</a> for merchants that your integration serves.</p>
```

Comment:
To refetch, is it to send a request to the endpoint again?
`GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/transactions`